src/Compiler/TypedTree/QuotationPickler.fs

```diff
@@ -254,7 +254,7 @@ module SimplePickle =
     when 'T:not null
 #endif
     > =
-        { tbl: HashMultiMap<'T, int> // This should be "Dictionary"
+        { tbl: HashMultiMap<'T, int when 'T: not null> // This should be "Dictionary"
```

Comment:
Hi @progressive-galib - this won't compile :)
'T is already defined and constrained up above, this should hold for `tbl` - I don't think changes here are needed.